README.md

```diff
@@ -46,3 +46,19 @@ If you don't have Flakes support:
 ```bash
 nix-shell
 ```
+
+## Note for metals/bloop users
```

Comment:
**thought:** this and the above contributing notes should eventually go to a `CONTRIBUTING` file. One Day!